Composer/packages/server/src/models/asset/assetManager.ts

```diff
@@ -18,7 +19,19 @@ import { copyDir } from '../../utility/storage';
 import StorageService from '../../services/storage';
 import { IFileStorage } from '../storage/interface';
 import { BotProject } from '../bot/botProject';
-import { runYeomanTemplatePipeline } from '../../workers/templateInstallation.worker';
+import { templateGeneratorPath } from '../../settings/env';
+import { BackgroundProcessManager } from '../../services/backgroundProcessManager';
+
+let workerPath = '@bfc/server-workers/lib/templateInstallation.worker';
+if (isElectron) {
+  workerPath = path.join(
+    process.resourcesPath,
+    'app.asar.unpacked/node_modules/@bfc/server-workers/lib/templateInstallation.worker'
+  );
+}
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires, security/detect-non-literal-require
+const { runYeomanTemplatePipeline } = require(workerPath);
```

Comment:
nit: this can be await import(path) right before it's used to avoid the linter complaints.